index.src.html

```diff
@@ -330,11 +333,11 @@ <h3 id="queue-report" algorithm>
 
   5.  If |settings| is given, then
 
-      1.  Let |environment| be |settings|'s <a>realm execution context</a>'s
-          <a spec=ecmascript>realm</a>'s <a spec=ecmascript lt=realm>ECMAScript
-          global environment</a>.
+      1.  Let |scope| be |settings|'s <a>realm execution context</a>'s
+          <a spec=ecmascript>realm</a>'s <a spec=ecmascript>global object</a>.
```

Comment:
This should just be `|setting|'s [=environment settings object/global object=]`

Reply:
I would have thought so, but [HTML§8.1.3.2](https://html.spec.whatwg.org/multipage/webappapis.html#environment-settings-objects) doesn't define it, and [HTML§8.1.3.3](https://html.spec.whatwg.org/multipage/webappapis.html#realms-settings-objects-global-objects) explains this roundabout way of getting to it.

Reply:
I'm not sure I understand the response... HTML's https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-global is meant to be something other specs can reference when they want to go from ESOs to global objects.

Reply:
Ahh, now I'm reading it right -- I didn't realize that the text at the end was a definition; I thought that it was explaining how to get to the global object by going through the Realm.

I was looking for something like "An _environment settings object_ has a **global object**, which is..."

Reply:
Done!